DEVELOPING.md

```diff
@@ -0,0 +1,98 @@
+This file contains tips to help you take (and understand) your first steps in
+the world of Theia development.  Are you in a hurry?  See the [tl;dr](#tldr).
+
+# How to build Theia and the example applications
+
+Theia is a framework to build IDEs, so you can't really "run" Theia itself.
+However, you can run the example applications included in its repository.  One
+is a browser-based IDE and the other is the electron-based equivalent.
+
+## Clone the repository
+
+    $ git clone https://github.com/TypeFox/Theia.git
+
+The directory containing the Theia repository will now be referred to as
+`$THEIA`, so if you want to copy-paste the examples, you can set the `THEIA`
```

Comment:
Indeed, it was the intent, to avoid having to do things like `cd ../examples/browser` and `cd ../electron`, which break if you decide to skip the "build the browser example" step, for example.